### PR TITLE
add eip155-3912.json add drac.json

### PR DESCRIPTION
**Pull request by DRAC-Network**
add DRAC Chain
Update _data/chains/eip155-3912.json
Update _data/chains/drac.json

add new network, include icon RPC website  chainId explorers

### CODE REVIEW

File: _data/icons/drac.json [hunk 1]
```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://bafybeihogmgrzljritbv2sdkoj3r34iiueg2oyby6uibk36uznuar4bvuu",
```
Comment (ligi) [inline _data/icons/drac.json:3]:
```suggestion
    "url": "ipfs://QmXbsQe7QsVFZJZdBmbZVvS6LgX9ZFoaTMBs9MiQXUzJTw",
```

**Comment by DRAC-Network** [comment]
modify eip155-3912.json
ShortName must match this pattern: '^[A-Za-z0-9-_]{1,64}$'

**Review by ligi — CHANGES_REQUESTED** [review]
Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json)
e.g. run `npx prettier --write _data/*/*.json`

**Comment by DRAC-Network** [comment]
> Please run the prettier to format your json according to the style [defined here ](https://github.com/ethereum-lists/chains/blob/master/.prettierrc.json) e.g. run `npx prettier --write _data/*/*.json`

It has been modified and updated as required, please check for confirmation.Thank you.

**Review by ligi — APPROVED** [review]

**Pull request merged**